tutorials/histfactory/example.py

```diff
@@ -82,7 +83,102 @@ def main():
     meas.PrintXML( "xmlFromPy", meas.GetOutputFilePrefix() );
 
     # Now, do the measurement
-    ROOT.RooStats.HistFactory.MakeModelAndMeasurementFast( meas );
+    ws = ROOT.RooStats.HistFactory.MakeModelAndMeasurementFast(meas)
+
+    # Retrieve the ModelConfig
+    modelConfig = ws.obj("ModelConfig")
+
+    # Extract the PDF and global observables
+    pdf = modelConfig.GetPdf()
+    globalObservables = ROOT.RooArgSet(modelConfig.GetGlobalObservables())
+
+    '''
+    parameters in globalObservables:
+
+    nominalLumi
+    nom_alpha_syst1 -> was set as constant in the beginning
+    nom_alpha_syst2
+    nom_alpha_syst3
+    nom_gamma_stat_channel1_bin_0
+    nom_gamma_stat_channel1_bin_1
+
+    '''
+
+    # Perform the fit
+    result = pdf.fitTo(ws.data("obsData"), ROOT.RooFit.Save(), ROOT.RooFit.PrintLevel(-1), ROOT.RooFit.GlobalObservables(globalObservables))
+
+    # Getting list of Parameters of Interest and getting first from them
+    poi = modelConfig.GetParametersOfInterest().first()
+
+    nll = pdf.createNLL(ws.data("obsData"))
+    profile = nll.createProfile(poi)
+
+    # frame for future plot
+    frame = poi.frame()
+
+    frame.SetTitle("")
+    frame.GetXaxis().SetTitle("-log likelihood")
+    frame.GetYaxis().SetTitle(poi.GetTitle())
```

Comment:
```suggestion
    frame.GetYaxis().SetTitle("-log likelihood")
    frame.GetXaxis().SetTitle(poi.GetTitle())
```
This should be the other way around